docs/dev/roadmap/code_structure/class_outlines/main_c.md

```diff
@@ -0,0 +1,5 @@
+# main.c
```

Comment:
I think individual files might be too granular (unless the idea is to only create these files to document key decisions)?

My worry would be these getting left out of date with the actual files and turning into more misleading docs

Reply:
I'd say that is a valid concern. My idea was to only create files for essential systems. Such as: absolute starting point (main,) file system, UI system, etc. A file would describe the broad, conceptual design choices for a subsystem which would allow for consistent implementation of subclasses, not necessarily detailed code explanations